rotkehlchen/api/v1/schemas.py

```diff
@@ -806,6 +807,7 @@ class ModifiableSettingsSchema(Schema):
         # Check that all values are unique
         validate=lambda data: len(data) == len(set(data)),
     )
+    pnl_order = fields.String(load_default=None)
```

Comment:
change name. Also just use a `SerializableEnumField`. Will also verify and include choices

Reply:
Ok